controllers/discordactions.js

```diff
@@ -466,10 +479,18 @@ const getUserDiscordInvite = async (req, res) => {
       return res.boom.notFound("User invite doesn't exist");
     }
 
-    return res.json({
-      message: "Invite returned successfully",
-      inviteLink: invite?.inviteLink,
-    });
+    if (dev) {
+      return res.json({
+        message: "Invite returned successfully",
+        inviteLink: invite?.inviteLink,
+        purpose: invite?.purpose,
+      });
+    } else {
+      return res.json({
+        message: "Invite returned successfully",
+        inviteLink: invite?.inviteLink,
+      });
+    }
```

Comment:
use early return

Reply:
what do you mean by early return ?

Reply:
early return-> you return for true condition and do not write else block for it.

Reply:
done